README.md

```diff
@@ -50,6 +50,8 @@ If you are looking for a good place to start, check out the [good first issue la
 
 ### Quickstart
 
+If you are interested in running your own instance of Atlas, please take a look at our [operator guide](docs/operator-guide.md).
```

Comment:
I think running as operators, as distinct from working on the code base, is such a different, and arguably even more relevant, use-case to the typical reader, that it deserves its own section.

Reply:
Sure, elevated to separate section